synapse/lib/modelrev.py

```diff
@@ -877,7 +883,10 @@ async def save():
                             continue
 
                         subcurv = sode['props'].get(subprop.name)
-                        if subcurv == subnorm: # pragma: no cover
+                        if subcurv is not None:
```

Comment:
do we want to compare the `(valu, stortype)` tuples here in case we need to update the stortype?